controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -431,6 +473,35 @@ func checkHttpMonitorExists(dynatraceApiClient *dynatrace.DynatraceApiClient, ho
 	return false, nil
 }
 
+func getClusterRegion(hostedcontrolplane *hypershiftv1beta1.HostedControlPlane) (string, error) {
+	if hostedcontrolplane == nil {
+		return "", fmt.Errorf("hostedcontrolplane is nil %v", hostedcontrolplane)
+	}
+
+	clusterRegion := hostedcontrolplane.Spec.Platform.AWS.Region
+	if clusterRegion == "" {
+		return "", fmt.Errorf("aws region is not set in hcp %v", hostedcontrolplane)
+	}
+
+	return clusterRegion, nil
+}
+
+func determineDynatraceClusterRegionName(clusterRegion string, monitorLocationType string) (string, error) {
+	//public
+	if monitorLocationType == "PublicAndPrivate" {
+		return getDynatraceEquivalentClusterRegionName(clusterRegion)
+	} else if monitorLocationType == "Private" {
```

Comment:
Can we use the [hypershift-defined values](https://github.com/openshift/hypershift/blob/main/api/hypershift/v1alpha1/hostedcluster_types.go#L1042-L1056) instead of hardcoded strings?

Reply:
Thanks! Now we're using `hypershiftv1beta1.PublicAndPrivate` and `hypershiftv1beta1.Private` instead of "PublicAndPrivate" and `"Private"`.